setup/gcp/installer/forseti_installer.py

```diff
@@ -122,15 +121,28 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             bool: Whether or not the deployment was successful
             str: Deployment name
         """
-        deployment_name, return_code = gcloud.create_deployment(
+        deployment_name = gcloud.create_deployment(
             self.project_id,
             self.organization_id,
             deployment_tpl_path,
             self.config.installation_type,
-            self.config.datetimestamp,
+            self.config.timestamp,
             self.config.dry_run)
 
-        deployment_completed = not return_code
+        # Status tracker function, will return true if the deployment is done.
```

Comment:
nit: drop this comment, as our code should be self-documenting enough that we don't need comments to explain what the code is doing (i.e. we only need to document `why` the code is doing something).

Reply:
Updated.